Authenticator/Source/OTPAppDelegate.swift

```diff
@@ -49,13 +47,8 @@ class OTPAppDelegate: UIResponder, UIApplicationDelegate {
         // Restore white-on-black style
         SVProgressHUD.setDefaultStyle(.Dark)
 
-        let rootViewController = TokenListViewController(viewModel: tokenList.viewModel,
-            delegate: tokenList)
-        tokenList.presenter = rootViewController
-        let navController = UINavigationController(rootViewController: rootViewController)
-        navController.navigationBar.translucent = false
-        navController.toolbar.translucent = false
-
+        let navController = AppViewController(viewModel: appModel.viewModel, actionHandler: appModel)
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 101 characters (line_length)